relations/src/proc_macro/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "liminal-ark-relation-macro"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
I think we should bump `minor` version here instead, if we want to follow semver rules, since we're breaking the interface for all of the downstream consumers of the macro -- i.e. they have to add `()` to the attributes.